deploy/reimburse.ts

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs'
+import { contract, deploy } from 'ethereum-mars'
+import { TrustToken } from '../build/artifacts'
+import { utils } from 'ethers'
+
+const readCsv = (text: string) => text.split('\n').map(row => row.split(' ') as [string, string]).slice(0, -1)
+
+const reimbursementData = readCsv(fs.readFileSync('./deploy/data/tfTUSDReimbursements.csv').toString())
+
+deploy({}, () => {
+  const tru = contract('trustToken_proxy', TrustToken, { skipUpgrade: true })
```

Comment:
Proxy contract has changed a bit since it was deployed so I had to cheat a bit and tell mars that TRU is under the proxy address